### ISSUE TITLE
Editing a PK-Sim module

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#1408]
Describe the steps that should be performed when user wants to edit a PK-Sim module.

- [ ] Warning that this would exclude the module from qualification
- [x] Change the status of the module to "extension module".
- [ ] Delete the snapshot
- [x] If the snapshot is not assigned to any other modules, delete it.

**Comment by PavelBal** [comment]
No, the steps that should be implemented in the Re-Qualification projects.

**Comment by rwmcintosh** [comment]
The current warning does not mention requalification.
<img width="590" height="161" alt="Image" src="https://github.com/user-attachments/assets/4ad07455-c1e3-4769-8936-01abf2c56493" />

We can add something like 

> The module will not be rebuilt in PK-Sim during requalifications if it is converted to an extension module. Continue?

We currently do not unassign the snapshot and we still allow the user to rebuild or export the snapshot from the context menu.

<img width="487" height="275" alt="Image" src="https://github.com/user-attachments/assets/f37f1ffa-fd3f-441a-879c-16238169a98f" />

A new PK-Sim module is created.

I think we decided to keep this as a feature.

@Yuri05 @msevestre @PavelBal
